src/dmd/backend/cg.d

```diff
@@ -52,3 +50,110 @@ Symbol *tls_get_addr_sym;       // function __tls_get_addr
 int TARGET_STACKALIGN = 2;      // default for 16 bit code
 int STACKALIGN = 2;             // varies for each function
 }
+
+import core.stdc.stdio;
+
+extern (C++) __gshared
```

Comment:
Does it have to be `extern(C++)` ?

Reply:
No. But as my usual practice, I change one thing at a time, for reasons explained many times.